20181004-demo-sprint-4.md

```diff
@@ -8,14 +8,60 @@ Slides: [ZDS2 Demo Spint 4.pdf](/community/bestanden/zds2-demo-sprint-4.pdf)
 
 ## Feedback
 
-Alle reacties tijdens de demo zijn hieronder opgenomen, in de categoriën 'proces' en 'inhoudelijk'. Waar nodig zijn issues aangemaakt. Dit zijn de antwoorden van het scrum team.
+Alle reacties en vragen tijdens de demo zijn hieronder opgenomen. Waar nodig zijn issues aangemaakt. Dit zijn de antwoorden van het scrum team. Reacties zijn uiteraard van harte welkom.
 
+#### Waarom moeten bij het aanmaken van een ObjectInformatieobject (zaakdocument) twee calls gedaan worden naar de DRC (Te weten eentje voor het aanmaken van een Informatieobject en eentje voor het maken van het relatieobject ObjectInformatieobject (het oude zaakdocument)).
+Dit komt omdat in het relatieobject de verwijzing naar zowel de zaak als het informatieobject opgenomen moet worden dus de moeten eerst bestaan. NB Mogelijk zou dit in één gecombineerde "convenience method" opgenomen kunnen worden.
 
-### Proces
+#### Wanneer een document bij een besluit hoort, moet dit dan zowel aan de zaak als het besluit gekoppeld worden? Deze opzet betekent dat de componenten (ZRC en DRC) elkaar moeten kunnen bereiken? 
+Dat klopt inderdaad. Mogelijk spelen hier ook nog authenticatie en autorsatie issues mee, dit zullen we moeten bekijken. Autorisatie en Authenticatie staat voor de komende sprint (5) op het programma.
 
-#### [Aan te vullen...]
+#### Moet deze invulling extra eisen stellen aan het compliant zijn?
+Dat klopt inderdaad, waar voorheen compliancy werd vastgesteld op berichtuitwisseling worden nu ook eisen gesteld aan de architectuur en hoe de berichten verwerkt worden.Dit is een bekend probleem met microservices wat opgelost kan worden met NLX. Theo Peters (VNG) concludeert dat een applicatie niet alleen moet voldoen aan de API maar ook aan gebruik van NLX.
 
+#### Conflicteert het dubbel vastleggen van de relatie niet met het CG principe van eenmalige opslag?
+Nee, het informatieobject wordt nog steeds maar eenmaal opgelagen. Alleen de relatie met andere objecten wordt meermalen opgeslagen, dit zjn echter verwijzingen.
```

Comment:
Dan conflicteert het dubbel opslaan van de relatie toch? Is dat minder erg omdat het 'verwijzingen' zouden zijn? Vind ik een vreemde redenering. Een relatie is ook een gegeven. 
@sergei-maertens @joeribekker Wat is de echte reden? Staat die wellicht in onze ontwerp-keuzes?

Reply:
Het hele design hiervan staat in de ontwerpkeuzes. Het is een trade-off, en de insteek is dat inderdaad enkel de verwijzing naar de (plek van de ) relatie opgenomen is, niet de relatie met meta-gegevens zelf.